iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -193,8 +198,12 @@ iXBRLViewer.prototype._checkDocumentSetBrowserSupport = function () {
 }
 
 iXBRLViewer.prototype.load = function () {
-    var iv = this;
-    var inspector = this.inspector;
+    const iv = this;
+    const inspector = this.inspector;
+
+    if (!this.isViewerEnabled()) {
```

Comment:
This disables the viewer as mentioned in #571 , however, it was a miscommunication on my part. I considered the "viewer" to be everything around the original document. What I intended to request was to disable everything except the `viewer-pane`. 

The goal would be to still load the original html file with no inspector or top bar.

Reply:
That should be exactly what this does.  What happens when you try this?

Reply:
It resulted in a blank page when I tried. It looks like it will never enter the `setTimeout` since it does the return when disabled.

Reply:
That'd explain it.  We should probably make it throw an error or just ignore the option completely if you try to use it in stub viewer mode.

Reply:
That sounds like a good idea and would be less confusing overall IMO.

I switched the gui plugin to not use stub viewer and things worked as expected.

Tangentially, it seems like [this](https://github.com/Arelle/ixbrl-viewer/blob/dc8dcde75239693e83f0ad796eae996556190e2f/iXBRLViewerPlugin/iXBRLViewer.py#L437C19-L437C19) should be updated to `ixbrlviewer.html` instead? Otherwise, the plugin opens to a 404 not found due to the missing `i`.

Reply:
Thinking some more, I think we can just make this work in stub viewer mode, as we can just make it redirect to the iXBRL document.  Done in a492818

> Tangentially, it seems like [this](https://github.com/Arelle/ixbrl-viewer/blob/dc8dcde75239693e83f0ad796eae996556190e2f/iXBRLViewerPlugin/iXBRLViewer.py#L437C19-L437C19) should be updated to ixbrlviewer.html instead? Otherwise, the plugin opens to a 404 not found due to the missing i.

Yes, I think that would make sense, but I think we should tackle it separately as I'm not sure what else might be relying on that naming convention.

Reply:
👍 , might just want to update the PR description that stub viewer is supported now for posterity.

Reply:
Good call - done.